scripts/troubleshooting/hs-manifest-gather/script.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -e
+set -o nounset
+set -o pipefail
+
+# shellcheck source=/dev/null
+source /managed-scripts/lib/sftp_upload/lib.sh
+
+# Define expected values
+TODAY=$(date -u +%Y%m%d)
+DUMP_DIR="/tmp/${CLUSTER_ID}_manifests_${TODAY}"
+TARBALL_NAME="${CLUSTER_ID}_manifests_${TODAY}.tar.gz"
+TARBALL_PATH="${DUMP_DIR}/${TARBALL_NAME}"
+
+# Create dump dir
+mkdir -p "${DUMP_DIR}"
+
+# Function to collect ACM HyperShift must-gather - manifests only
+collect_manifests() {
+
+  # Check if required CLIs are present
+  if ! command -v oc >/dev/null 2>&1; then
+    echo "oc binary not found in PATH"
+    exit 1
+  fi
+  if ! command -v jq >/dev/null 2>&1; then
+    echo "jq binary not found in PATH"
+    exit 1
+  fi
+  if ! command -v yq >/dev/null 2>&1; then
+    echo "yq binary not found in PATH"
+    exit 1
+  fi
+
+  # Sanity check to make sure desired cluster namespace exists
+  hcp_ns_list_length=$(oc get namespace -l api.openshift.com/id="${CLUSTER_ID}" -o json | jq -r '.items | length')
+  if [ "${hcp_ns_list_length}" -ne 1 ]; then
+    echo "Number of HCP namespaces matching ${CLUSTER_ID} must be 1, ${hcp_ns_list_length} found"
+    exit 1
+  fi
+
+  # Fetch the target HCP namespace for the cluster name
+  hc_namespace=$(oc get namespace -l api.openshift.com/id="${CLUSTER_ID}" -o json | jq -r '.items[0].metadata.labels["kubernetes.io/metadata.name"]')
+
+  # Fetch the HostedCluster name
+  hc_name=$(oc get HostedCluster.hypershift.openshift.io -l api.openshift.com/id="${CLUSTER_ID}" -n "${hc_namespace}" -o json | jq -r '.items[0].metadata.name')
+
+  
+  # TODO(ACM-16170): replace this with an official ACM release image once it's available
+  # local acm_hypershift_image="registry.redhat.io/rhacm2/acm-must-gather-rhel9:v2.11"
```

Comment:
I can see the offcial image is available when I tested to pull from my local:
```
podman pull registry.redhat.io/rhacm2/acm-must-gather-rhel9:v2.11
Trying to pull registry.redhat.io/rhacm2/acm-must-gather-rhel9:v2.11...
Getting image source signatures
Checking if image destination supports signatures
Copying blob e7dc8c72147d done   | 
Copying blob 8fcd1353fe74 done   | 
Copying config 29eb776b06 done   | 
Writing manifest to image destination
Storing signatures
29eb776b06a0988c91c9a96c9c64191f919cb0498df13cdd6f44febd026e44bb
```
Should we use the official image instead?